lib/KCard/index.vue

```diff
@@ -107,19 +127,20 @@
        * @param {String} value - The layout value.
        * @returns {Boolean} - True if the value is not empty, false otherwise.
        */
-      layout: {
-        type: String,
-        default: 'horizontal',
+       layout: {
+      type: String,
+      default:  Layouts .HORIZONTAL,
+      validator: cardValidator(Layouts, 'layout')
       },
       /**
        * Controls how the thumbnail appears in the card.
        * Expected Options: 'none' (default), 'small', or 'large'.
        * */
-      thumbnailDisplay: {
-        type: String,
-        required: false,
-        default: 'none',
-      },
+       thumbnailDisplay: {
+      type: String,
+      default: Thumbnail_Displays.NONE,
```

Comment:
It'd be ideal if we could use constants as `default` values, but unfortunately there's limitation regarding our documentation generation that can't pick it up. Would you please revert to `default: 'none'` just on this line?

Same applies to `default:  Layouts .HORIZONTAL,`

This will ensure that consumers will understand what values, previously mentioned in the prop description, are used as defaults


![Screenshot from 2024-07-31 21-27-36](https://github.com/user-attachments/assets/69627e41-1009-4620-8ae6-c45d4ce87204)

Reply:
It'd be ideal if we could use constants as `default` values, but unfortunately there's limitation regarding our documentation generation that can't pick it up. e?

![Screenshot from 2024-07-31 21-27-36](https://github.com/user-attachments/assets/69627e41-1009-4620-8ae6-c45d4ce87204)


Would you please revert to `default: 'none'` on this line? Same applies to `default:  Layouts .HORIZONTAL,`

This will ensure that consumers will understand what values, previously mentioned in the prop description, are used as defaults